extension/test/integration/integration-test-set-up.js

```diff
@@ -31,19 +50,30 @@ function _overrideApiExtensionBaseUrlConfig(apiExtensionBaseUrl) {
   module.exports = config
 }
 
-async function initServerAndExtension({ ctpClient, ctpProjectKey }) {
-  await initServer()
+async function initServerAndExtension(
+  { ctpClient, ctpProjectKey },
+  isServerAuthEnabled
+) {
+  await initServer(ctpProjectKey, isServerAuthEnabled)
   await initExtension(ctpClient, ctpProjectKey)
 }
 
-async function initServer() {
+async function initServer(ctpProjectKey, isServerAuthEnabled) {
   const port = config.getModuleConfig().port || 8000
   server = serverBuilder.setupServer(routes)
   // note: ngrok should be restarted for every test case, otherwise there will be
   // 429 Too Many Requests error. This is due to the limit of maximum opened HTTP connections,
   // which is 40 connections at the same time as we're using Free program (https://ngrok.com/pricing).
   const apiExtensionBaseUrl = await ngrok.connect(port)
   _overrideApiExtensionBaseUrlConfig(apiExtensionBaseUrl)
+  console.log('######')
+  console.log(ctpProjectKey)
+  console.log(isServerAuthEnabled)
+  if (isServerAuthEnabled && ctpProjectKey) {
+    console.log('add config')
+    _addAuthObjectToServerConfig(ctpProjectKey)
+  }
+  console.log('no config')
```

Comment:
Seems like you put a lot of console.log here 😄

Reply:
because i was checking something that not happened in local machine, so i added debug logs 
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/a121d31a68bde6a8c25b36cb710f2ff3c5e7911a